core-js/src/test/javascript/cdf/components/TextInputComponent-spec.js

```diff
@@ -55,5 +70,29 @@ define([
 
       dashboard.update(textInputComponent);
     });
+
+    /**
+     * Checks if process event is triggered on every key press if the property refreshOnEveryKeyUp is true
+     */
+    it("process change on each key up event", function(done) {
+      textInputComponent.refreshOnEveryKeyUp = true;
+
+      spyOn(textInputComponent.dashboard, "processChange").and.callFake(function(name) {
+        expect(name).toBe(textInputComponent.name);
+        delete textInputComponent.refreshOnEveryKeyUp;
+        done();
+      });
+
+      dashboard.update(textInputComponent);
+
+      spyOn(textInputComponent, "_setCursor").and.callFake(function() {
```

Comment:
This test was broken. @afrjorge fixed it and I already cherry picked the changes.